CartMgmt/Cart/cart.css

```diff
@@ -0,0 +1,84 @@
+@import url(https://fsharpui.netlify.app/main.css);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --white-color: #fff;
+  --grey-color: #808080;
+  --peach-color:rgb(241, 203, 219);
+
+}
+.navigation {
+  margin: 0rem;
+  width: 100%;
+  background-color: var(--peach-color);
+}
+
+.icon-heart,
+.icon-cart {
+  font-size: 3rem;
+}
+
+.cart-item {
+  flex: 50;
+}
+
+.cart-price {
+  margin: 5rem;
+  flex: 50;
+}
+
+.price-container {
+  align-items: center;
+  background-color: var(--peach-color);
+  border-radius: 5rem;
+  border-color: antiquewhite;
+  margin-right: 30rem;
+  margin-top: 20rem;
+  padding: 3rem;
+}
+
+.btn-shopping-cart:hover {
+  color: black;
+}
+
+table {
+  width: 50%;
+  border-collapse: collapse;
+}
+
+.quantity {
+  text-align: center;
+  display: inline-block;
+  font-size: 1.7rem;
+  height: 2rem;
+  width: 2rem;
+  border: 1px solid var(--grey-color);
+  border-radius: 50%;
+}
+.container-flex{
+margin-bottom: auto;
+border-color: none;
+}
+
+hr.someClass {
+  background-color: black;
+  width: 50%;
+  height: 2px;
+  border-top: 1px solid black;
```

Comment:
use hexacodes for the colors and have it in variables

Reply:
ok